canary-release/cfexample.yaml

```diff
@@ -0,0 +1,17 @@
+title: Canary Release with Kubernetes
+description: canary release pattern implementation with Codefresh pipelines and K8S
+sourceUrl: https://github.com/Codefresh-Examples/Examples/canary-release
+baseUrl: https://github.com/Codefresh-Examples/Examples/tree/master/canary-release
```

Comment:
this also won't work. just use: `https://raw.githubusercontent.com/Codefresh-Examples/Examples/master/canary-release`